enhancements/declarative-index-config.md

```diff
@@ -200,51 +205,56 @@ $ cat etcd.json
             "image": "quay.io/coreos/etcd-operator@sha256:c0301e4686c3ed4206e370b42de5a3bd2229b9fb4906cf85f3f30650424abec2"
         }
     ]
-},
+}
 {
     "schema": "olm.bundle",
-    "name": "etcdoperator.v0.9.2-clusterwide", 
+    "name": "etcdoperator.v0.9.2-clusterwide",
     "package": "etcd",
     "image": "quay.io/operatorhubio/etcd:v0.9.2-clusterwide",
     "version": "0.9.2-clusterwide",
     "properties":[
         {
-            "name": "olm.package",
-            "values": {
+            "type": "olm.package.provided",
+            "value": {
                 "packageName": "etcd",
                 "version": "0.9.2-clusterwide"
             }
         },
         {
-            "name":"olm.gvk",
-            "type": "provided",
-            "values":{
-                "group": "etcd.database.coreos.com", 
-                "kind": "EtcdBackup", 
+            "type": "olm.gvk.provided",
+            "value": {
+                "group": "etcd.database.coreos.com",
+                "kind": "EtcdBackup",
                 "version": "v1beta2"
             }
         },
         {
-            "name": "skipRange",
-            "value": ">=0.9.0 <0.9.2"
+            "type": "olm.skipRange",
+            "value": ">=0.9.0 <=0.9.2-0"
+        },
+        {
+            "type": "olm.skips",
+            "value" : "etcdoperator.v0.6.0"
         },
```

Comment:
As this EP is currently written, `skips` and `skipRange` are "global" bundle properties and apply to all channels. Now that we're exposing the upgrade graph directly to catalog maintainers, is it possible to boil `replaces`, `skips`, and `skipRange` down to a list of explicit `upgradeFrom` edges that are channel specific?

Reply:
And another related question. If we can't do the above, should `skips` and `skipRange` be channel-specific?

Reply:
@joelanford fyi I had the same thoughts during the EP review process, maybe these conversation that stemmed from a proposal to express explicit `upgradeFrom` edges and channel specific `skips`/`skipRange` can provide more context: 

https://github.com/operator-framework/enhancements/pull/37#issuecomment-746975161
https://github.com/operator-framework/enhancements/pull/37#discussion_r554506459